src/TodoList.js

```diff
@@ -16,10 +16,10 @@ const todoList = [
     },
   ];
```

Comment:
It's cut off in this snippet but the entire `todoList` array can be deleted as it is no longer being used